src/pages/home/report/ReportActionItemMessageEdit.tsx

```diff
@@ -81,20 +89,25 @@ function ReportActionItemMessageEdit(
     const theme = useTheme();
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
+    const containerRef = useRef<View>(null);
     const reportScrollManager = useReportScrollManager();
     const {translate, preferredLocale} = useLocalize();
     const {isKeyboardShown} = useKeyboardState();
     const {isSmallScreenWidth} = useWindowDimensions();
     const prevDraftMessage = usePrevious(draftMessage);
-
+    const suggestionsRef = useRef<SuggestionsRef>(null);
+    const mobileInputScrollPosition = useRef(0);
+    const cursorPositionValue = useSharedValue({x: 0, y: 0});
+    const tag = useSharedValue(-1);
```

Comment:
What is this used for? The name is very generic so I'm having a hard time understanding.

Reply:
We use the tag to store the ID of the text input. Later, we use it in onSelectionChange to pick up the proper text input data. I left a comment